frontend/packages/pipelines-plugin/src/components/repository/conts.ts

```diff
@@ -0,0 +1,23 @@
+interface RepositoryLabel {
+  labels: string[];
+  baseURL?: string;
+}
+
+export enum REPOSITORY_FIELDS {
+  REPOSITORY = 'Repository',
+  BRANCH = 'Branch',
+  REPO_URL = 'Repo url',
+  SHA = 'sha',
+  EVENT_TYPE = 'Event type',
+}
+
+export const repositoryLabels: Record<REPOSITORY_FIELDS, RepositoryLabel> = {
+  [REPOSITORY_FIELDS.REPOSITORY]: { labels: ['pipelinesascode.tekton.dev/repository'] },
+  [REPOSITORY_FIELDS.BRANCH]: { labels: ['pipelinesascode.tekton.dev/branch'] },
+  [REPOSITORY_FIELDS.REPO_URL]: {
+    labels: ['pipelinesascode.tekton.dev/url-org', 'pipelinesascode.tekton.dev/url-repository'],
+    baseURL: 'https://github.com',
```

Comment:
Move this out as a separate enum called `SupportedVCS` and add `github` in there

Reply:
moved out baseURL